framework/src/config/mod.rs

```diff
@@ -1,9 +1,22 @@
+use failure::Fail;
```

Comment:
We should put it on our `todo` to rip this stuff out and use [clap](https://github.com/clap-rs/clap) instead and integrate this with the config-rs structures we already using. Low priority though.

Reply:
Will do and agree